assets/css/main.css

```diff
@@ -0,0 +1,125 @@
+body {
+  font: 100% Helvetica, sans-serif;
+  background: url(/assets/images/bg.jpg) no-repeat center center fixed;
+  background-size: cover;
+  margin: 0;
+  padding: 0;
+}
+
+.menu-item {
+  margin-left: 3rem;
+  margin-right: 3rem;
+  font-size: 2rem;
+}
+
+.menu-list {
+  /* display: -webkit-box;
+  display: -ms-flexbox; */
```

Comment:
- [ ] It's better to not commit commented codes, would you remove them? That will make your code to look much more cleaner